proto/notification/recipient.proto

```diff
@@ -19,8 +19,13 @@ option go_package = "github.com/bucketeer-io/bucketeer/proto/notification";
 
 message Recipient {
   enum Type { SlackChannel = 0; }
+  enum Language {
+    Japanese = 0;
+    English = 1;
```

Comment:
Can you set the `English` as the first option in the num? Because English is the default language in the system.